File: src/codes/Users_Manual/gain/plane_0.rs
# use autd3::prelude::*;
# fn main() {
# let nx = 0.;
# let ny = 0.;
# let nz = 0.;
# let _ = 
Plane {
    dir: UnitVector3::new_normalize(Vector3::new(nx, ny, nz)),
    option: PlaneOption {
        intensity: EmitIntensity::MAX,
        phase_offset: Phase::ZERO,
    },
};
# }

File: src/codes/Users_Manual/gain/plane_0.cpp
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
//~const auto nx = 0.0;
//~const auto ny = 0.0;
//~const auto nz = 1.0;
Plane(Vector3(nx, ny, nz),
      PlaneOption{
          .intensity = std::numeric_limits<Intensity>::max(),
          .phase_offset = Phase::zero(),
      });
//~return 0; }